CHANGES.rst

```diff
@@ -1,12 +1,12 @@
 Change History
 --------------
 
-1.0.1 (unreleased)
+1.1.0 (unreleased)
 ~~~~~~~~~~~~~~~~~~
 
-- Add support for Python 3.6, 3.7.
+- Add support for Python 3.6, 3.7, 3.8, 3.9.
 
-- Dropped Python 2.6 and 3.3 from Travis CI.
+- Drop support for Python 2.6 and 3.3.
```

Comment:
The changelog for 1.0.0 says it already dropped 3.3 and 2.6 support.  AFAICT the people who did this forgot to update .travis.yml.  And since .travis.yml is now gone entirely, I don't think we need to say anything about that in the changelog.

```suggestion
```